scripts/test_django.py

```diff
@@ -0,0 +1,702 @@
+                current_content = current_content.strip()
+                message = TestMessage(status=current_status, content=current_content, description=current_description)
+                self.messages.append(message)
+                in_message = False
+                current_content = ""
+                current_status = ""
+                current_description = None
+                in_header = False
+            # Otherwise, if we're in a message, we want to append what we're reading to whatever we're currently building
+            elif in_message:
+                # If we're in the header, the status will be the first encountered line, so if we're in a header 
+                # and we don't have a status, assign this line to the status
+                if in_header and not current_status:
+                    current_status = line.strip()
+                # If we're still in the header, we know we want to attach this to the description
+                elif in_header:
+                    # Since we've read a new line, we want to add a line separator to make sure that this data is 
+                    # separated from the previously read
+                    if current_description:
+                        current_description += os.linesep
+
+                    current_description += line
+                # Otherwise, we want to just tack the read data onto the content of the message we're building
+                else:
+                    # If we're on the final line of the messages, just tack on a new line to our content, 
+                    # otherwise attach the new content after a new line in order to keep the natural separation from stderr
+                    current_content += os.linesep if line == LAST_END_MESSAGE else f"{os.linesep}{line}"
+
+        # If it's detected that we're still parsing and have reached the end of the document, we want to attach 
+        # what we have as a new message
+        if current_content and in_message:
+            current_content = current_content.strip()
+            message = TestMessage(status=current_status, content=current_content, description=current_description)
+            self.messages.append(message)
+
+    def print(self, verbose: bool = None, quiet: bool = None):
+        """
+        Print the test results in one of a different number of formats
+
+        Args:
+            verbose: Whether to print in verbose mode
+            quiet: Whether to print in quiet mode
+        """
+        # verbose and quiet mode conflict, so throw an error if they are encountered
+        if verbose and quiet:
+            print(
+                "Output cannot be both quiet and verbose; choose either '--quiet' or '--verbose', but not both",
+                file=sys.stderr
+            )
+            # Exit with a code of -1 to indicate that this was an application error, not a test error or failure
+            exit(-1)
```

Comment:
`POSIX` exit codes are unsigned 8 bit integers. Fortunately, python is smart enough and wraps this around to `255`, but you may want to change this to some other meaningful positive number.